authorinomanifests/authorino.yaml

```diff
@@ -537,11 +622,10 @@ subjects:
   namespace: kuadrant-system
 ---
 apiVersion: rbac.authorization.k8s.io/v1
-kind: ClusterRoleBinding
+kind: RoleBinding
```

Comment:
If https://github.com/Kuadrant/kuadrantctl/pull/11/files#r671323761 is correct, this should be kept of `ClusterRoleBinding` kind.